map/framework.cpp

```diff
@@ -2472,7 +2477,9 @@ void Framework::SetRouterImpl(RouterType type)
 
     router.reset(
         new CarRouter(m_model.GetIndex(), countryFileGetter,
-                      SingleMwmRouter::CreateCarRouter(m_model.GetIndex())));
+                      SingleMwmRouter::CreateCarRouter(m_model.GetIndex(),
+                                                       bind(&RoutingSession::GetTrafficColoring,
```

Comment:
bind() generate an anonymous object and I pass it by const ref and keep it by const ref for using later. You're right. It should be a segfault. Thanks. Let's get read of bind at all in that place.